src/tasks/tests/tests.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Please don't check in solution files.

Reply:
Generally folks will want both tests and impl projects open in VS. what's the recommended way to do that - create a solution manually?